docs/Training-On-Your-Data.md

```diff
@@ -369,9 +369,50 @@ This will spin up a TPU VM instance and install Levanter on it. You can then run
 gcloud compute tpus tpu-vm ssh my-tpu   --zone us-east1-d --worker=all --command="WANDB_API_KEY=... levanter/infra/launch.sh python levanter/src/levanter/main/train_lm.py --config_path gs://path/to/config.yaml"
 ```
 
-### GPU
 
-TODO, but you can mostly follow the guide for [TPU](#tpu) above.
+### GPU Training
```

Comment:
I think this should go in the GPU guide actually. We should also say that if you're SLURM (and using Pyxis), you won't need to do provide the distributed arguments. JAX/Levanter will infer them for you.